LondonBritishLibrary/orient/BLorient818.xml

```diff
@@ -34,152 +38,765 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                             <idno>Wright 291</idno>
                         </altIdentifier>
                </msIdentifier>
-               
+
                     <msContents>
                         <summary/>
                         <msItem xml:id="ms_i1">
                             <locus from="3r" to="46v"/>
                             <title type="complete" ref="LIT1709Kebran"/>
+                            <textLang mainLang="gez"></textLang>
+                            <note>In this manuscript the text is not divided into
+                            chapters.</note>
+
+                            <msItem xml:id="ms_i1.1">
+                              <locus from="46v"></locus>
+                              <title type="complete" xml:lang="en">Index</title>
```

Comment:
I wonder whether the index is normally transmitted with the work and should have a div in the edition? (Maybe it has already and I didn't see it)

Reply:
I have checked the manuscripts, the index seems to be a later addition and is not transmitted in many manuscripts. So, I think that it would not be justified to add to the edition by now.